Sources/PasscodeKit/API.swift

```diff
@@ -103,10 +103,9 @@ public extension View {
     ///   - onCompletion: The closure to execute when the passcode was checked.
     func checkPasscode(
         isPresented: Binding<Bool>,
-        allowBiometrics: Bool = false,
```

Comment:
This option is there on purpose. When adding a `checkPasscode` in the app, often it is not wanted to be able to check with biometrics.

Example: The Passcode & Face ID Settings also require to input the passcode and this method is exactly that.

If the integrator wants to allow biometrics, they can always set this to `true`.